.github/workflows/pr-check.yml

```diff
@@ -15,3 +15,23 @@ jobs:
   unit-test:
     name: Unit Test
     uses: ./.github/workflows/ci-unit-test.yml
+
+  build-ok:
+    name: Build OK
+    needs: [lint, unit-test]
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo OK
+
+  test-ok:
+    name: Test OK
+    needs: [unit-test]
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo OK
+
+  publish-ok:
+    name: Publish OK
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "No-op on PRs to satisfy branch checks"
```

Comment:
These are the same steps Bors is enforcing (via `bors.toml`).

Enforcement is now done via GitHub's repo settings, so we now require them on PRs _as well as_ merge queues. Hence, we re-define them on the PR workflow and no-op where appropriate.

Reply:
Ack. We'll probably want to manage the repo settings in a checked-in-place (perhaps via an IaC solution of some kind 🤔) for Real projects.